.docker/Dockerfile.nginx-drupal

```diff
@@ -1,10 +1,18 @@
+##
+# `nginx-drupal` image
```

Comment:
What is the point of this comment? The file name already says that.

`Custom Nginx image for Drupal configuration` sounds more descriptive